### DISCUSSION TITLE
QSocketNotifier Error

### DISCUSSION THREAD

**Discussion by jkissi** [marcomusy/vedo#500] [Q&A] [answered]
Hi!!
I use the Spyder IDE in conda for my development and every time I close a plot window I have generated and the program finishes, I get a QSocketNotifier Error as shown. Every time I run a program the errors compound, so after launching Spyder, the first time I get one error, the second time I get 2 errors, and so on. The socket number is always different per launch but once I get one error, that number is always quoted See below:

![multiple_socket_edit](https://user-images.githubusercontent.com/8246894/139542996-9c0b1ba5-f3e2-4425-8a87-fda5cb3a0c71.png)

Problem is, these errors compound until I have so many that the kernel explodes and I have to restart it. I was wondering if you've ever encountered something like this and/or you might know how to stop it?

**Comment by marcomusy** [comment, marked as answer]
I never encountered this problem, and never be reported before afaik.. it might be an issue of qt/spyder? not really sure.
If you believe this is related to vedo please let me know.

**Reply by jkissi** [reply]
Perhaps it is a Qt thing...I believe Spyder uses that too but the error only seems to happen when I run the vedo visualisation tool, so I was hoping that maybe you might so idea of where a conflict could be...?

**Reply by marcomusy** [reply]
maybe this fixes it:
https://github.com/spyder-ide/spyder/issues/13275#issuecomment-708415063

**Reply by jkissi** [reply]
I tried this, but no it didn't work. Not to worry, I just thought I'd ask.
Cheers!
